src/index.css

```diff
@@ -0,0 +1,167 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
+
+.App {
+  width: 90%;
+  margin: auto;
+}
+
+.mainHeader {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  column-gap: 22px;
+  align-items: center;
+}
+
+.logo img {
+  width: 100px;
+  height: 100px;
+}
+
+/* nav {
+  padding-right: 9%;
+} */
+
+nav ul {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  column-gap: 12px;
+  align-items: center;
+}
+
+nav ul a {
+  text-decoration: none;
+  background-color: transparent;
+}
+
+nav ul a:focus {
+  text-decoration: underline;
+}
+
+nav ul a:active {
+  text-decoration: underline;
```

Comment:
The active page should be indicated with an underline ✔

Reply:
@mateo951 Thank you for the approval